zstacklib/zstacklib/utils/pci.py

```
@@ -38,9 +40,256 @@ def update_cache_devices(devices_dict):
+                            for device in data[key]:
+                                bus = device.get('pci_bus') or device.get('PCI Bus') or device.get('bus_id', '')
+                                if bus and pci_address.lower() in bus.lower():
+                                    return True
+            except (ValueError, TypeError, KeyError):
+                # Fallback: check if PCI address is in raw output
+                pass
+            # Fallback: check if PCI address is in output
+            if pci_address.lower() in o.lower():
+                return True
+    except Exception:
+        pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)